### ISSUE TITLE
Unexpected Behavior: ``Model.query`` does not respect updated values

### ISSUE THREAD

**Issue by rileycrane** [coleifer/walrus#10]
Updating an attribute of an object does not affect the results of a query for that object.  Here is a minimal example demonstrating the unexpected behavior.

python shell
```
from walrus import *
db = Database(host='127.0.0.1', port=6379, db=0)
class Test(Model):
    database     = db
    myattr = BooleanField(index=True)

# Create an object
test = Test(myattr=True)
# Query for it
list(Test.query(Test.myattr==True)) # [<Test: 1>]
# Update the attribute
test.myattr=False
test.save()

# Query again - UNEXPECTED RESULT
list(Test.query(Test.myattr==True)) # [<Test: 1>]
test.myattr # False


```

**Comment by coleifer** [comment]
Verified this with a test-case, looking into what might be going on. I suspect this is query caching, but I'll look.

**Comment by coleifer** [comment]
Thanks so much for reporting this. Bug is fixed and I've pushed release 0.2.2 containing the fix.

**Comment by rileycrane** [comment]
awesome! thanks!
